autotest/pst_tests.py

```diff
@@ -819,6 +819,25 @@ def new_format2_test(tmp_path):
         raise Exception("fail:"+str(e))
     os.chdir(b_d)
 
+def new_format_pestpp_options_test(tmp_path):
```

Comment:
Don't think `tmp_path` is needed in this instance. This is a pytest thing and is useful if modifying files, to prevent test interference. Here just reading the reference file so should be safe without (providing that files isn't getting beaten up by another test).